### PR TITLE
Feature/niad 839 gp2 gp deploy vp cluster

### PR DESCRIPTION
**Pull request by LukaszKwasniewski**

### CODE REVIEW

File: terraform/aws/components/gp2gp/module_wiremock_ecs_service.tf [hunk 1]
```diff
@@ -0,0 +1,60 @@
+module "gp2gp_wirmock_ecs_service" {
```
Comment (LukaszKwasniewski) [inline terraform/aws/components/gp2gp/module_wiremock_ecs_service.tf]: wirmock != wiremock

**Comment by LukaszKwasniewski** [comment]
Notes:
* Please add the variable pointing to wiremock lb to gp2gp env variables
* Please add a .terraform-version file with version 0.14.7 - gp2gp has been deployed to vp with the latest terraform version
* add a variable to control if wiremock should be created, or not, `count = var.create_wiremock ? 1 : 0` at the top of ecs service module declaration.

File: terraform/aws/components/gp2gp/module_wiremock_ecs_service.tf (hunk 1 above)
Comment (RIashfaqahmed) [reply terraform/aws/components/gp2gp/module_wiremock_ecs_service.tf]: fixed

File: terraform/aws/components/gp2gp/locals_env_variables.tf [hunk 2]
```diff
@@ -55,6 +55,10 @@ locals {
     {
       name = "GP2GP_GPC_HOST"
       value = var.gp2gp_gpc_host
+    },
+      {
```
Comment (LukaszKwasniewski) [inline terraform/aws/components/gp2gp/locals_env_variables.tf]: remove the two spaces before {

Comment (RIashfaqahmed) [reply terraform/aws/components/gp2gp/locals_env_variables.tf]: fixed

File: terraform/aws/components/gp2gp/module_wiremock_ecs_service.tf (hunk 1 above)
Comment (RIashfaqahmed) [reply terraform/aws/components/gp2gp/module_wiremock_ecs_service.tf]: Fixed

**Review by RIashfaqahmed — APPROVED** [review]

**Pull request merged**